roles-and-groups.md

```diff
@@ -0,0 +1,153 @@
+- **Approve the nomination**: All maintainers should vote on that nomination and the vote passes when a majority of maintainers vote to proceed. The voting procedure is explained in the same template [vote_pr_template.md](https://github.com/hiero-ledger/governance/blob/main/.github/PULL_REQUEST_TEMPLATE/vote_pr_template.md).
+- **Make adjustments**: A maintainer is considered inactive if there has been no activity for six months and there has been no response when trying to contact them.
+
+Maintainers are responsible for keeping track of project Issues and Discussions and help pushing inactive conversations. It is a responsibility of the maintainers to close Issues and Discussions that are considered outdated or stale to the current goals of the project.
+
+Project maintainers or the TSC may remove a maintainer who:
+
+- Breaks Code of Conduct.
+- Is inactive for 6 months and does not respond when contacted.
+- Keeps making bad decisions (their contributions are negatively affecting the project's performance).
+
+More on maintainer guidelines can be found on the [**LFDT TAC site**](https://lf-decentralized-trust.github.io/governance/governing-documents/MAINTAINERS-file.html).
+
+### Voting
+
+When it comes to electing and voting for new committers or new maintainers, the project should consider the following:
+
+For candidates seeking promotion to Committers or Maintainers:
+- The candidate needs to show participation in the project by contributing with code, issues, discussions, reviews, etc.
+- These contributions need to show proper understanding of the project and quality of the work.
+- 3 to 6 months worth of contributions is considered a good minimum for candidates to be selected for a role promotion.
+- A Committer or Maintainer will be considered inactive if they shown no activity for 6 months at least, in such case, they could be selected for removal.
+- A Committer can maintain their project status independently from their employment.
+
+For Maintainers and Committers voting on new candidates:
+- When a candidate is selected for becoming a Committer or a Maintainer, this proposal needs to be made via PR request against the config.yaml in the governance repo.
+- An example of a promotion can be found in the following [PR](https://github.com/hiero-ledger/governance/pull/176).
+- Once the PR is created, all the maintainers of the repo in question need to be added as reviewers and their vote should be provided.
+- The creator of the PR should notify all maintainers that a vote is now open.
+- Maintainers can vote in favor of the candidate's promotion by approving the PR with a comment indicating approval.
+- Maintainers can vote against the candidate's promotion by posting a comment in the PR along with their explanation.
+- Maintainers can abstain from voting by posting a comment in the PR along with their explanation.
```

Comment:
> Maintainers can abstain from voting

Minor note: we will want to clarify the verbiage here, and above; e.g. 'All Maintainers are required to participate in a nomination, but may effectively abstain from voting by only posting a comment in the PR.'

Reply:
done

Reply:
Thank you1